<?xml version="1.0"?>
<launch>

    <arg name="dev" default="/dev/ttyACM1"/>
    <arg name="baud" default="38400"/>
    <arg name="address" default="129"/>
    <arg name="max_speed" default="1.7"/>
    <arg name="ticks_per_meter" default="89478.2"/>
    <arg name="base_width" default="0.2485"/>
    <arg name="run_diag" default="true"/>

    <node if="$(arg run_diag)" pkg="roboclaw_node" type="little_helper_node_2.py" name="roboclaw_node_2">
        <param name="~dev" value="$(arg dev)"/>
        <param name="~baud" value="$(arg baud)"/>
        <param name="~address" value="$(arg address)"/>
        <param name="~max_speed" value="$(arg max_speed)"/>
        <param name="~ticks_per_meter" value="$(arg ticks_per_meter)"/>
        <param name="~base_width" value="$(arg base_width)"/>
    </node>

    <node pkg="diagnostic_aggregator" type="aggregator_node"
          name="diagnostic_aggregator_2">
        <rosparam command="load"
                  file="$(find roboclaw_node)/config/roboclaw_diag.yaml"/>
    </node>

</launch>

